CONTRIBUTING.md

```diff
@@ -144,6 +144,21 @@ Here is a recommended way to get setup:
 6. Write code, open a PR from your branch when you're ready
 7. If you need to rebase your fork's PR branch onto master to resolve conflicts: `git fetch upstream`, `git rebase upstream/master` and force push to Github `git push --force origin your-branch`
 
+## Changelog Conventions
+
+What warrants a changelog entry?
+
+- Any change that introduces changes to the public API *must* have a changelog entry.
+- Any change that affects user security *must* have a changelog entry
+- Any user-facing or visually impacting change *must* have a changelog entry
+- Any performance improvement *should* have a changelog entry
+- Any bugfix *should* have a changelog entry
+- Any contribution from a community member *may* have a changelog entry, no matter how small
+- Any documentation related changes *should not* have a changelog entry
+- Any regression change introduced and fixed within the same release *should not* have a changelog entry
+- Any internal refactoring, technical debt reduction, render test or benchmark related change *should not* have a changelog entry
+- Any changelog entry should be descriptive and concise; it should explain the change to a reader without context
```

Comment:
I would condense some of these into one lines — not need to be too explicit since this is hard to read, e.g.:

- Any change that affects the public API, visual appearance or user security ...
- Any bug fix or a performance improvement ...

Reply:
Yep that's a good point, tried to condense for conventions that were similar